all-repos.yaml

```diff
@@ -64,6 +65,7 @@
 - https://github.com/psf/black
 - https://github.com/IamTheFij/ansible-pre-commit
 - https://github.com/IamTheFij/docker-pre-commit
+- https://github.com/macisamuele/language-formatters-pre-commit-hooks
```

Comment:
these intentionally aren't alphabetized -- they're ordered by time-of-addition and then grouped by organization (so this should be added at the end)

Reply:
GTK - i'll update

Reply:
pushed this fix to put at the end